### DISCUSSION TITLE
"I2CScan":"Error 5 at 0x01 (bus2)"

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#22402] [Q&A] [answered]
i try to get i/o modules (PCF8574, MCP23xx, PCA9685) running on i2c bus 2.
it is a custom build based on tasmota32s3-qio_opi-120

``` 
00:18:48.425 CMD: i2cscan
00:18:48.443 RSL: RESULT = {"I2CScan":"Device(s) found on bus1 at 0x5d"}
00:18:48.495 RSL: RESULT = {"I2CScan":"Error 5 at 0x01 (bus2)"}
```

i tried with each module. together or seperatly, even without a module connected. it returns that error.
What does that mean? i found nothing about this error .

**Comment by s-hadinger** [comment, marked as answer]
This error means that the bus is stuck. Something is driving the lines low and Tasmota can't use the bus anymore

**Comment by fapoo** [comment]
Hi, i have the same issue with this [board](https://templates.blakadder.com/ESP32_Relay_X2.html). Can't read any i2c device. Is there a way to fix it?

**Reply by sfromis** [reply]
A typical cause of that error is bad wiring for the I2C stuff.

**Reply by fapoo** [reply]
Well the devices were directly connected to the pins + an analogue in. But i will try it again. Thanks.
